v63005/outref/gtm8980.txt

```diff
@@ -0,0 +1,66 @@
+# Running v63005/gtm8980.csh to test that rare uses of VIEW and $VIEW are handled correctly 
+
+# Create a 3 region DB with gbl_dir mumps.gld and regions DEFAULT, AREG, and BREG
+
+
+# GTM-8980 calls for testing calls of VIEW and $VIEW with an empty string
+# This test case is already handled by r122/viewcmdfunc subtest and will be skipped
+# (in test6^viewcmdfunc and test7^viewcmdfunc)
+
+# Run test1^gtm8980.m
+Testing that $VIEW("STATSHARE") returns the correct values
+--------------------------------------------------------------------------
+
+VIEW "STATSHARE" 
+$VIEW("STATSHARE"): 
+1
+
+VIEW "NOSTATSHARE" 
+$VIEW("STATSHARE"): 
+0
+
+VIEW "STATSHARE":"AREG" 
+$VIEW("STATSHARE"): 
+2
+
+
+# Run test2^gtm8980.m
+Test $VIEW("STATSHARE","<region>") for selectively disabled region
+--------------------------------------------------------------------------
+
+VIEW "NOSTATSHARE" 
+VIEW "NOSTATSHARE":"BREG" 
+$VIEW("STATSHARE","BREG"): 
+0
+
+
+# Run test3a^gtm8980.m, test3b^gtm8980.m, test3c^gtm8980.m
+Test $VIEW("GARBAGE PARAMETER") error message for parameter name
+--------------------------------------------------------------------------
+
+$VIEW("GARBAGE PARAMETER"): 
+%GTM-E-VIEWNOTFOUND, View parameter GARBAGE PARAMETER not valid
+		At M source location test3a+5^gtm8980
+
+GTM>
+
+Test VIEW "GARBAGE PARAMETER" error message for parameter name
+--------------------------------------------------------------------------
+
+VIEW "GARBAGE PARAMETER" 
```

Comment:
GARBAGE PARAMETER does not test what we want. This test will run the same even in V63004 and V63005. Instead, you need to run a test like the following.
```
view "GVFILE":"DEFAULT"
```
GVFILE is a view parameter that is valid with $VIEW but not with VIEW. This will show GVFILE in the error message in V63005 whereas it did not show GVFILE in V63004.